0-good-advice/notes/what-i-wish-someone-had-told-me.md

```diff
@@ -0,0 +1,72 @@
+Read it yourself: https://blog.samaltman.com/what-i-wish-someone-had-told-me
+
+Tool to help you apply it to your life: https://chat.openai.com/g/g-uPMOViqnT-altman-advisor
+
+
+## Applying Advice to Different Stages Of Business
+
+### 1. Ideation Stage (No Idea to Concept)
+Key Elements:
+- Harnessing optimism, obsession, self-belief.
+- Embracing raw horsepower and personal connections.
+- Focusing on audacious ideas that matter.
+
+Checklist:
+- [ ] Do I feel optimistic and deeply passionate about my concept?
+- [ ] Am I leveraging my strengths and connections for this idea?
+- [ ] Is my idea challenging yet impactful and motivating?
+
+### 2. Planning Stage (Concept to Initial Strategy)
+Key Elements:
+- Long-term orientation and planning.
+- Concentrating resources on high-conviction bets.
+- Clear, concise communication.
+
+Checklist:
+- [ ] Have I aligned my strategy with long-term goals?
```

Comment:
doesn't really communicate benefit if you say yes or no to each one. Feels better than actually helping.